tests/Library/Core/ArrayTranslateTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author Patrick Kelly <patrick.k@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license Proprietary
+ */
+
+namespace VanillaTests\Library\Core;
+
+use VanillaTests\SharedBootstrapTestCase;
+
+/**
+ * Tests for arrayTranslate().
+ */
+class ArrayTranslateTest extends SharedBootstrapTestCase {
```

Comment:
Since this is a library test w/ no container usage, or addon startup, I'd recommend using the PHPunit base test case.

`SharedBootstrapTestCase` brings along a lot of baggage with it (starting session, db connection, sql driver, addon autoloader, etc).

You should be able to just use `PHPUnit\Framework\TestCase`. for cases where you still need the container, our `MinimalContainerTestCase` would be appropriate too. I'll be overviewing this Friday during my lunch & learn in more detail.